Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -678,6 +678,93 @@ public void CLIDynamicJSON_OutputsPass_Virtual_AllCustomizedTypes_Test()
             Assert.AreEqual(WorkSpace.Instance.RunsetExecutor.Runners[1].BusinessFlows[0].RunStatus, eRunStatus.Passed, "Flow_C- Validating BF execution Passed");
         }
 
+        /// <summary>
+        /// Testing JSON existing Runset with multi Runners and StopOnFailure turn on
+        /// </summary>   
+        [TestMethod]
+        public void CLIDynamicJSON_Existing_StopRunnersOnFailure_StopOn_Test()
+        {
+            // Arrange
+            string jsonConfigFilePath = CreateTempJSONConfigFile(Path.Combine(TestResources.GetTestResourcesFolder("CLI"), "CLIDynamicJSON_Existing_StopRunnersOnFailure_StopOn.Ginger.AutoRunConfigs.json"), mSolutionFolder);
+
+            // Act            
+            CLIProcessor CLI = new CLIProcessor();
+            CLI.ExecuteArgs(new string[] { "dynamic", "-f", jsonConfigFilePath });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5700269)